project_user_product/README.rst

```diff
@@ -0,0 +1,21 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Project user product
+====================
+
+* With this module can be defined in projects "user-products". When imputed in
+  a task, if the user that imputed has a line in "users-products" of projects,
+  to take the cost price of that product, if it does not have a line in
+  "users-products", leave standard functionality.
+
```

Comment:
De esta forma creo que se entiende mejor:
This module allows to define "user-products" in projects. When there is a task line, if the user who has generated it has a "users-products" line in projects, it would take the cost price of the product otherwise it would take the normal price.